dev/com.ibm.ws.jca.cm/src/com/ibm/ejs/j2c/MCWrapper.java

```diff
@@ -517,6 +518,20 @@ public void setSharedPoolCoordinator(Object sharedPoolCoordinator) {
         this.sharedPoolCoordinator = sharedPoolCoordinator;
     }
 
+    /**
+     * Set the unshared pool coordinator
+     */
+    @Override
+    public void setUnSharedPoolCoordinator(Object unSharedPoolCoordinator) {
+        this.unSharedPoolCoordinator = unSharedPoolCoordinator;
+    }
+
+    @Override
+    public Object getUnSharedPoolCoordinator() {
+        // TODO Auto-generated method stub
```

Comment:
Small nit, we should avoid unnecessary TODOs so the valid ones are more apparent